commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java

```diff
@@ -210,7 +210,6 @@ private String buildEditorLayout(WebuiRequestContext context) throws Exception {
             .append("CKEDITOR.plugins.addExternal('confirmBeforeReload','/commons-extension/eXoPlugins/confirmBeforeReload/','plugin.js');")
```

Comment:
This line should be moved to eXoConfig.js to make the plugin available.

Reply:
done